plugins/mcreator-localization/lang/texts.properties

```diff
@@ -3310,4 +3320,4 @@ notification.update_available.msg=There is a new version of MCreator available.<
 notification.patch_available.msg=There is a more recent version of {0} available.<br>\
   <br>Installed version: {0}.{1}\
   <br>New version: {0}.{2}
-notification.plugin_updates.msg=There are some plugin updates available
\ No newline at end of file
+notification.plugin_updates.msg=There are some plugin updates available
```

Comment:
This change here is not needed. Make sure this line is not the last line of the file. The last line must be an empty line.

Reply:
You'd have to change the file on your local branch, then push using Intellij or Git commands. If you're editing online from GitHub, it adds a newline by default. Don't worry too much about it for the time being, though

Reply:
Ok